e2e/fixtures/ssr-target-bundle/README.md

```diff
@@ -0,0 +1,3 @@
+# SSR Bundling for NPM packages with environment depended code parts
+
+Choosing the right code based on the `export` section of the `react-textarea-autosize` module based on the backend server deployment target ('node' | 'webworker') and the client browser. No browser only code should be bundled with server only code.
```

Comment:
To me, it's not our fault that when target=webworker, the bundler picks the browser version, instead of worker version. So, it's our temporarily workaround.

Reply:
What would you suggest?

Reply:
Nothing. Sorry for the confusion. It's just a note so that we are on the same page.